app/src/main/java/com/fish/monsters/features/homeScreen/HomeScreen.kt

```diff
@@ -0,0 +1,96 @@
+package com.fish.monsters.features.homeScreen
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Surface
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.drawWithContent
+import androidx.compose.ui.graphics.Brush
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.DpSize
+import androidx.compose.ui.unit.dp
+import com.fish.monsters.R
+import com.fish.monsters.common.views.CapText
+import com.fish.monsters.common.views.FishButton
+import com.fish.monsters.common.views.OutlinedFishButton
+import com.fish.monsters.core.theme.FishMonstersTheme
+
+@Composable
+fun HomeScreen() {
+    val surfaceColor = MaterialTheme.colorScheme.surface
+    Column(horizontalAlignment = Alignment.CenterHorizontally) {
+        Image(
+            painter = painterResource(R.drawable.kraken_home_screen),
+            contentDescription = "",
+            contentScale = ContentScale.FillWidth,
+            modifier = Modifier
+                .fillMaxWidth()
+                .drawWithContent {
+                    drawContent()
+                    drawRect(
+                        Brush.verticalGradient(
+                            colors = listOf(
+                                Color.Transparent,
+                                surfaceColor
+                            )
+                        )
+                    )
+                }
```

Comment:
`drawWithContent()`  pozwala na narysowanie co chcemy nad lub pod widokiem do którego dajemy ten `Modifier`. `drawContent()` rysuje ten widok w którym się znajdziemy (czyli dla nas to ten `Image`). `drawRect()` rysuje kwadrat z podanym przez nas `Color` lub `Brush`. w naszym przypadku to `Brush` z gradientem który zrobiliśmy na szkoleniu. Zrobiłem to w ten sposób żeby gradient był zawsze dopasowany do zdjecia które przypominam ma dynamiczne wymiary zależne od wielkości ekranu.